spec/features/data_access/status_page_spec.rb

```diff
@@ -114,9 +110,8 @@
   context 'when the current user tries to access an invalid retrieval ID' do
     before :all do
       Capybara.reset_sessions!
-      load_page :search, overlay: false
-      login
-      load_page '/data/retrieve/not_a_valid_retrieval_id'
+      load_page :search, authenticate: 'edsc'
```

Comment:
Does `load_page` not handle a path? I'm assuming this is doing this to authenticate the user?

Reply:
`load_page` can do paths, but I couldn't do this on with authenticate and have it actually try to load the page, so I authenticate first, then visit the page